versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx

```diff
@@ -36,3 +38,11 @@ error: the server doesn't have a resource type "sliceconfigs"
 - Ensure you are on the controller cluster to run the commands: `kubectx -c`.
 - Export the configuration file using this command: `export KUBECONFIG=kubeslice/<path-to-the-kubeconfig-file>`.
 
+
+### Getting an Unverified Developer Error message on macOS
+When you try to install kubeslice-cli on macOS, you get the `Unverified Developer Error Message`.
+This error message appears when you try to install an application from a developer who has not registered with Apple.
```

Comment:
... when you try to install an application from a developer who has not been registered with Apple  or .. who is not registered with Apple.

Reply:
updated